pkgs/by-name/qm/qmplay2/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  lib,
+  alsa-lib,
+  callPackage,
+  cmake,
+  ffmpeg,
+  fribidi,
+  game-music-emu,
+  libXdmcp,
+  libXv,
+  libass,
+  libcddb,
+  libcdio,
+  libpulseaudio,
+  libsidplayfp,
+  libva,
+  libxcb,
+  ninja,
+  pkg-config,
+  qt5,
+  qt6,
+  stdenv,
+  taglib,
+  vulkan-headers,
+  vulkan-tools,
+  # Configurable options
+  qtVersion ? "6", # Can be 5 or 6
+}:
+
+let
+  sources = callPackage ./sources.nix { };
+in
+assert lib.elem qtVersion [ "5" "6" ];
+stdenv.mkDerivation (finalAttrs: {
+  pname = sources.qmplay2.pname + "-qt" + qtVersion;
+  inherit (sources.qmplay2) version src;
+
+  postPatch = ''
+    pushd src
+    cp -va ${sources.qmvk.src}/* qmvk/
+    chmod --recursive 744 qmvk
+    popd
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+  ]
+  ++ lib.optionals (qtVersion == "6") [
+    qt6.wrapQtAppsHook
+  ]
+  ++ lib.optionals (qtVersion == "5") [
+    qt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    alsa-lib
+    ffmpeg
+    fribidi
+    game-music-emu
+    libXdmcp
+    libXv
+    libass
+    libcddb
+    libcdio
+    libpulseaudio
+    libsidplayfp
+    libva
+    libxcb
+    taglib
+    vulkan-headers
+    vulkan-tools
+  ]
+  ++ lib.optionals (qtVersion == "6") [
+    qt6.qt5compat
+    qt6.qtbase
+    qt6.qtsvg
+    qt6.qttools
+  ]
+  ++ lib.optionals (qtVersion == "5") [
+    qt5.qtbase
+    qt5.qttools
+  ];
+
+  strictDeps = true;
+
+  # Because we think it is better to use only lowercase letters!
+  # But sometimes we come across case-insensitive filesystems...
+  postFixup = ''
```

Comment:
Is there a reason this is `postFixup` rather than `postInstall`?

Reply:
A long decade ago I thought it was not an installation thing.